airflow.cfg

```diff
@@ -3,9 +3,6 @@
 default_timezone = utc
 log_filename_template = {{ ti.dag_id }}/{{ ti.task_id }}/{{ execution_date.strftime("%%Y-%%m-%%dT%%H:%%M:%%S") }}/{{ try_number }}.log
 
-# The home folder for airflow, default is ~/airflow
-airflow_home = $AIRFLOW_HOME
```

Comment:
why are we rm'ing this?

Reply:
This was suggested by the warnings in the logs -- AIRFLOW_HOME is the source of truth in this case. I don't have a link to the particular issue for this one.